Desktop/widgets/factorlevellistbase.cpp

```diff
@@ -56,6 +54,8 @@ void FactorLevelListBase::bindTo(const Json::Value& value)
 	}
 	
 	_factorLevelsModel->initFactors(factors);
+	
+	BoundControlBase::bindTo(value);
```

Comment:
Why does it help?

Reply:
the `bindTo` call ultimately triggers `Json::Value FactorLevelListBase::createMeta()` in calculating the `boundValue` for this control. But it only does so once (unless the value changes of course) and is then cached.

See:
```
Json::Value FactorLevelListBase::createMeta()
{
	Json::Value meta(BoundControlBase::createMeta()); //Work from parent to get everything it defines
	
	auto factors = _factorLevelsModel->getFactors();
	
	if(factors.size() == 0)
		return meta;
	
	meta["encodeThis"] = Json::arrayValue;
	for (const auto & factor : factors)
	{
		meta["encodeThis"].append(factor.first);
		
		for(const auto & level : factor.second)
			meta["encodeThis"].append(level);
	}
	
	return meta;
}
```

If `bindTo` is called before `initFactors` the `auto factors = _factorLevelsModel->getFactors();` is pretty empty.